pom.xml

```diff
@@ -12,7 +12,7 @@
 		<module>core</module>
 		<module>eclipse</module>
 		<module>intellij</module>
-        <module>server</module>
+    <module>server</module>
```

Comment:
This does not look right.

Reply:
This is mostly an artifact of #283 which I would rather solve in a separate PR

Reply:
I am confused about the "corrected" indentation here.

Reply:
It is mostly an auto-format issue I suppose. I open these files in multiple editors and they all change indentation because we have not standardized yet. With #283 and editorconfig we can prevent these mistakes.